src/rastervision/tagging/tasks/predict.py

```diff
@@ -101,40 +101,82 @@ def compute_probs(run_path, model, options, generator, split):
                 flip_img = np.fliplr(img)
                 aug_batch.append(np.expand_dims(flip_img, axis=0))
                 aug_batch = np.concatenate(aug_batch, axis=0)
-
-                aug_probs = np.mean(model.predict(aug_batch), axis=0,
-                                    keepdims=True)
-
-                y_probs.append(aug_probs)
-        # Otherwise, predicting probs on unaugmented images by batch
-        else:
+                aug_probs = model.predict(aug_batch)
+
+                for aug_ind in range(NUM_AUG):
+                    y_probs[aug_ind].append(aug_probs[aug_ind])
```

Comment:
If you replace `aug_probs[aug_ind]` with `np.expand_dims(aug_probs[aug_ind], axis=0)` then you will have an array of shape [1, 17]. Then when you concatenate them, you'll have an array of shape [x, 17] and you won't need to enclose in [] as we were discussing.